### PR TITLE
DENG-432. Due documentation for the active_users_aggregates tables

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
This PR adds the documentation of the active_users_aggregates tables that we use build many core metrics and expose them in Looker.

It also aims to add a good practices to query views instead of tables.

### CODE REVIEW

**Review by gkabbz — APPROVED** [review]

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+[`telemetry_derived`](https://github.com/mozilla/bigquery-etl/tree/main/sql/moz-fx-data-shared-prod/telemetry_derived).
+
+## How to query
+
+| Dataset                             | BigQuery                                              | Looker                                                                                                           |
+|-------------------------------------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| active_users_aggregates             | Table `telemetry.active_users_aggregates`             | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates) |
+| active_users_aggregates_device      | Table `telemetry.active_users_aggregates_device`      | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_device)      |
+| active_users_aggregates_attribution | Table `telemetry.active_users_aggregates_attribution` | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_attribution) |
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:64]:
```suggestion
| `active_users_aggregates`             | Table `telemetry.active_users_aggregates`             | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates)             |
| `active_users_aggregates_device`      | Table `telemetry.active_users_aggregates_device`      | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_device)      |
| `active_users_aggregates_attribution` | Table `telemetry.active_users_aggregates_attribution` | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_attribution) |
```

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+The `active_users_aggregates_device` contains the metrics of
+daily, weekly and monthly active users, new profiles and search counts
+to analyze data per device.
+
+The reason to have this aggregate in addition to `active_users_aggregates`
+is to improve the query performance for final users, by separating the
+device analysis, which makes one of the biggest size columns in the table,
+as most devices have unique identifiers.
+
+#### Active users aggregates for attribution
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:32]:
```suggestion
#### Active users aggregates for attribution

```

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Contents
+
+#### Active users aggregates
+This aggregates contains the metrics daily, weekly and monthly active users,
+as well as new profile and search counts aggregated by Mozilla product and
+business dimensions: attribution parameters, channel, country, city, date,
+device model, distribution id, segment and OS details.
+
+#### Active users aggregates for device
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:22]:
```suggestion
#### Active users aggregates for device

```

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+The `active_users_aggregates` is a set of 3 tables designed to analyze clients
+activity on a daily basis, starting from the date of installation.
+These aggregates are created to support the migration of the GUD
+Growth and Usage Dashboard.
+
+Here is the link to the [proposal](https://docs.google.com/document/d/1qvWO49Lr_Z_WErh3I3058A3B1YuiuURx19K3aTdmejM/edit?usp=sharing).
+
+## Contents
+
+#### Active users aggregates
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:16]:
```suggestion
#### Active users aggregates

```

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+to analyze the source and context of client installations and the `cohorts`
+behaviour.
+It can be used to query the set attribution parameters in the context
+of the business core dimensions: country, submission_date, `app_name`
+and if the browser is set to default.
+
+The reason to have this aggregate in addition to `active_users_aggregates`
+is to improve the query performance for final users, by separating the
+analysis of the numerous attribution parameters, which is required with
+less regularity  than other dimensions and mostly for specific purposes.
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:44]:
```suggestion
less regularity than other dimensions and mostly for specific purposes.
```

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## How to query
+
+| Dataset                             | BigQuery                                              | Looker                                                                                                           |
+|-------------------------------------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| active_users_aggregates             | Table `telemetry.active_users_aggregates`             | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates) |
+| active_users_aggregates_device      | Table `telemetry.active_users_aggregates_device`      | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_device)      |
+| active_users_aggregates_attribution | Table `telemetry.active_users_aggregates_attribution` | [Explore](https://mozilla.cloud.looker.com/explore/combined_browser_metrics/active_users_aggregates_attribution) |
+
+## Looker visualizations with period over period analysis
```
Comment (badboy) [inline src/datasets/bigquery/active_users_aggregates/reference.md:66]:
```suggestion
## Looker visualizations with period over period analysis

```

**Review by badboy — CHANGES_REQUESTED** [review]
prettier is a bit more picky about these things.
I added suggestions for everything it complained about so you can apply that in the web UI.

File: src/datasets/bigquery/active_users_aggregates/reference.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+## Code Reference
+
+The query and metadata for the aggregates is defined in the corresponding
+sub-folder in bigquery-etl under
+[`telemetry_derived`](https://github.com/mozilla/bigquery-etl/tree/main/sql/moz-fx-data-shared-prod/telemetry_derived).
+
+## How to query
+
+| Dataset                             | BigQuery                                              | Looker                                                                                                           |
+|-------------------------------------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
```
[the remark this answers is not in the document]
Comment (lucia-vargas-a) [reply src/datasets/bigquery/active_users_aggregates/reference.md:64]: Oh wow, this one

**Review by lucia-vargas-a — COMMENTED** [review]
@badboy Thanks a lot for the review & suggestions!

**Review by badboy — APPROVED** [review]

**Pull request merged**